src/pyjlconv.jl

```diff
@@ -39,6 +39,25 @@ function awkwardobjectarray2jl(x::PyObject)
     OpaqueObjectArray{tp}(data)
 end
 
+function _conv_hist_edge(edge::AbstractVector{<:Real})
+    steps = diff(edge)
+    step_size = mean(steps)
+    if all(isapprox(first(steps)), step_size)
+        minimum(edge):step_size:maximum(edge)
+    else
+        edge
+    end
+end
+
+_hist_edges(edges::AbstractVector{<:Real}) = _conv_hist_edge(edges)
+_hist_edges(edges::NTuple{N,AbstractVector{<:Real}}) where N = map(_conv_hist_edge, edges)
+
+function roothist2jl(x::PyObject)
+    edges = _hist_edges(x.edges)
+    weights = x.values
+    Histogram(edges, weights)
```

Comment:
I know it's the default, but we could be over-precautious here and add `closed=:left` here, since ROOT defaults to this?

Reply:
Good point - I don't think the StatsBase default will change, but it certainly couldn't hurt.